.github/workflows/ci.yml

```diff
@@ -507,7 +506,6 @@ jobs:
       - name: Build OpenRCT2
         run: |
           . scripts/setenv -q && build -DCMAKE_CXX_COMPILER=clang++-20 -DCMAKE_BUILD_TYPE=Release -DAPPIMAGE=ON -DOPENRCT2_USE_CCACHE=on
-          strip bin/install/usr/bin/openrct2
```

Comment:
I noticed the app image barely changed its size, while linux builds went from ~90MB to ~150MB. Maybe it is already stripping it with the `build-appimage` ?

Reply:
Is this solved with the last force push? On the smartphone app it is trickier to check the binary sizes

Reply:
Appimage went up by 0.5MB, other Linux builds went up by ~60MB. I'm very happy with the increase in appimage size, maybe I can still look a bit into the other builds. I'll split the PR for now

Reply:
I didn't notice it earlier, but there was `-g` being added in our build commands. This flag actually takes an argument and defaults to `-g2` if none present. `-g2` generates most of the debug info, `-g0` (or no `-g`) does not produce any debug info, while `-g1` produces minimal information to figure out stack traces. I've changed to use `-g1` now

Reply:
Ah, makes sense. So I assume the builds other builds didn't see the size bump then.

Would it still make sense to not use `-g` at all on official releases to save on space? Or is it negligible?

Reply:
It's negligible, like 0.5MB. This gives us enough data to figure out stack traces but not source lines or any of the variables. Sometimes it's enough to figure out where the issue might be. If it isn't enough, we can increase the debug level and because we use immutable build environments and have reproducible builds, we can generate debug symbols for a build after it has crashed